### DISCUSSION TITLE
Importing Asset within Entry doesn't allow me to specify title for the asset

### DISCUSSION THREAD

**Discussion by chadcrowell** [engram-design/FeedMe#1022] [Ideas]
Using Craft CMS 3.6.5.1 and Feed Me 4.3.5.1.

I'm importing from a JSON feed where images are also being brought in with a title, and potentially a secondary description. The secondary description is a custom field attached to the asset. The title is the built in title field for assets. I see on the docs for Asset importing that titles can be specified, however, when importing an entry, I don't seem to be able to specify a title for the asset field within the entry, even though I can specify the custom field secondary description. See my screenshot

[https://d.pr/i/rqSCxs](https://d.pr/i/rqSCxs)

I've re-set this feed up several times and I'm not seeing anything that I am missing for this need.

**Comment by antcooper** [comment]
+1

I have this problem too. I'd rather be able to set the Title than create a secondary Caption field as I do now.

**Comment by BenMenking** [comment]
I might call this a bug instead of an enhancement.  In earlier versions of Craft this worked.  Here's a quick patch that got me rolling again.  In vendor/craftcms/feed-me/src/web/twig/variables/FeedMeVariable.php

```
+++ web/twig/variables/FeedMeVariable.php       2021-02-25 14:54:01.098697683 +0000
@@ -318,6 +318,7 @@
             'craft\fields\PositionSelect',
             'craft\fields\Radio',
            'craft\fields\Redactor',
+           'craft\redactor\Field'
         ];

         return in_array($class, $supportedSubFields, true);

```

My only guess is that when Redactor was removed from Craft and "plugin-ized" the class name changed but wasn't updated in this file.

**Comment by dgsiegel** [comment]
> I might call this a bug instead of an enhancement. In earlier versions of Craft this worked. Here's a quick patch that got me rolling again. 

That only seems to work for Redactor fields, but not for the title as originally requested.
